### PR TITLE
 [2fa] feature/CAES-181: updated 2fa flow

### PR DESCRIPTION
**Pull request by ipopov4xxi**
For now two factor is mandatory.

### CODE REVIEW

File: src/Controller/Api/TwoFactorAuthController.php [hunk 1]
```diff
@@ -73,102 +62,34 @@
...
-     *     methods={"DELETE"}
-     * )
-     *
-     * @param Request                      $request
-     * @param EntityManagerInterface       $manager
-     * @param GoogleAuthenticatorInterface $twoFactor
-     *
-     * @return FormInterface|JsonResponse
-     */
-    public function disable(Request $request, EntityManagerInterface $manager, GoogleAuthenticatorInterface $twoFactor)
```
Comment (aburov-4xxi) [inline src/Controller/Api/TwoFactorAuthController.php]: How i can disable 2FA?

File: src/Form/Request/TwoFactoryAuthEnableType.php [hunk 2]
```diff
@@ -20,7 +20,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ->add('secret', TextType::class, [
                 'property_path' => 'googleAuthenticatorSecret',
             ])
-            ->add('auth_code', TextType::class, [
+            ->add('authCode', TextType::class, [
```
Comment (aburov-4xxi) [inline src/Form/Request/TwoFactoryAuthEnableType.php]: snake_case for attribute is more readable into json

File: src/Controller/Api/TwoFactorAuthController.php (hunk 1 above)
Comment (ipopov4xxi) [reply src/Controller/Api/TwoFactorAuthController.php]: Enable always for now.

File: src/Form/Request/TwoFactoryAuthEnableType.php (hunk 2 above)
Comment (ipopov4xxi) [reply src/Form/Request/TwoFactoryAuthEnableType.php]: On project start was arrangement about camelCase. And all old stuff corresponds to it.

File: src/Controller/Api/TwoFactorAuthController.php (hunk 1 above)
Comment (aburov-4xxi) [reply src/Controller/Api/TwoFactorAuthController.php]: I think we should have optional this functionality
@beshkenadze confirm, please

File: src/Form/Request/TwoFactoryAuthEnableType.php (hunk 2 above)
Comment (aburov-4xxi) [reply src/Form/Request/TwoFactoryAuthEnableType.php]: camelCase for property on backend or class name is ok, but this is frontend attribute into form, but ok this is not critical

**Pull request merged**